message.go

```diff
@@ -171,24 +201,18 @@ func (tm *TaskMessage) Encode() (string, error) {
 
 // ResultMessage is return message received from broker
 type ResultMessage struct {
-	ID        string        `json:"task_id"`
-	Status    string        `json:"status"`
-	Traceback interface{}   `json:"traceback"`
-	Result    interface{}   `json:"result"`
-	Children  []interface{} `json:"children"`
+	Result interface{} `json:"result"`
+	Error  string      `json:"error"`
```

Comment:
Why do we need to make this error a string?

Reply:
Because interface doesnt support marshalling and unmarshalling